src/LLVM/Pretty.hs

```diff
@@ -399,7 +404,7 @@ ppFunctionArgumentTypes :: Type -> Doc
 ppFunctionArgumentTypes FunctionType {..} = ppParams pp (argumentTypes, isVarArg)
 
 ppCall :: Instruction -> Doc
-ppCall Call { function = Right f,..}
+ppCall inst@(Call { function = Right f,..})
```

Comment:
This looks like it might be leftover from debugging? I don’t think it actually changes anything.